v-next/hardhat-node-test-runner/src/task-action.ts

```diff
@@ -67,7 +79,21 @@ const testWithHardhat: NewTaskActionFunction<TestActionArguments> = async (
   }
 
   const tsx = new URL(import.meta.resolve("tsx/esm"));
-  process.env.NODE_OPTIONS = `--import "${tsx.href}"`;
+  const imports = [tsx.href];
+
+  if (hre.globalOptions.coverage === true) {
+    const coverage = new URL(
+      import.meta.resolve("@nomicfoundation/hardhat-node-test-coverage"),
+    );
+    imports.push(coverage.href);
+    process.env.HARDHAT_NODE_TEST_COVERAGE_DIR_PATH = path.join(
+      os.tmpdir(),
```

Comment:
I think we have a helper for this in the `utils/fs`, if not, we should.

Reply:
We don't yet, but it definitely makes sense to have one.

Reply:
I exposed the new utility here: https://github.com/NomicFoundation/hardhat/pull/6595/commits/f08b2fa9b32d58ecc28c785a7828e2b1225e5f61